client/src/main/java/org/apache/karaf/client/Main.java

```diff
@@ -142,14 +142,28 @@ public String getUpdatedPassword(ClientSession session, String prompt, String la
                     }
                 });
             }
+            
+            if (config.getUser()==null || config.getUser().isEmpty()) {
+            	while (true) {
+            		String user = console.readLine("Enter user : ");
```

Comment:
In order to be consistent with the password prompt, I will use `"Enter user:  "`.